scripts/README.md

```diff
@@ -54,21 +54,20 @@
 
 - <span><b>chairman.sh</b></span> <br/>  Run the Chairman consensus checker against a cluster started
                      by `shelley-testnet*.sh` family of scripts
-- <span><b>generator.sh</b></span> <br/> Run the transaction generator against a cluster started
```

Comment:
Generator was removed by @CodiePP  (there is a new `cardano-benchmarking` repo) so I also removed `Run transaction generator` - line 180